docs(content/update): migration guide Process Instance Modification (#1689)
camunda/camunda-docs-manual: content/update/minor/721-to-722/_index.md

1. For administrators and developers: [Sending telemetry feature removed](#sending-telemetry-feature-removed)
1. For administrators: [Database transaction isolation level `READ_COMMITTED` is enforced](#database-transaction-isolation-level-read-committed-is-enforced)
1. For developers: [Quarkus 3.14 Extension Update](#quarkus-3-14-extension-update)

> 1. For administrators and developers: [Cockpit modify filter changes](#cockpit-modify-filter-changes)

Reviewer: ❌ Broken link

Clicking on this link does not point to the respective section.
Most likely it is the result of renaming the title of the section and the title still points to the old id.

Let's adjust this to point to a functional link.

> 1. For administrators and developers: [Cockpit modify filter changes](#cockpit-modify-filter-changes)

Reviewer: The link is broken. Does the suggestion represent what you are trying to point to? Otherwise the title of the section needs to be renamed to `Cockpit Modify Filter Changes`
```suggestion
1. For administrators and developers: [Process Instance Modification Filter Changes](#process-instance-modification-filter-changes)
```

For a detailed guide on the new Quarkus properties, visit the updated [Quarkus Configuration]({{< ref "/user-guide/quarkus-integration/configuration.md" >}}) page.

To read more on how Quarkus extensions treat configuration differently, see this [Quarkus Migration Guide](https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.14#for-extension-developers) and the [Mapping Configuration to Objects Guide](https://quarkus.io/guides/config-mappings).

# Process Instance Modification Filter Changes

> In the Camunda Cockpit, the [Process Definition View][process-instance-modification] dialog has been updated to use the Historic API. With this change, the start date filter is now available within the workflow, while also aligning the Modify Workflow functionality with the Batch Operation API in terms of filter behavior.

Reviewer: ```suggestion
The [Process Definition View][process-instance-modification] dialog of Camunda Cockpit has been updated to use the Historic API. With this change, the `start date` filter is now available within the workflow, while also aligning the Modify Workflow functionality with the Batch Operation API in terms of filter behavior.
```

> In the Camunda Cockpit, the [Process Definition View][process-instance-modification] dialog has been updated to use the Historic API. With this change, the start date filter is now available within the workflow, while also aligning the Modify Workflow functionality with the Batch Operation API in terms of filter behavior.

Reviewer: 💭 `while also aligning the Modify Workflow functionality with the Batch Operation API in terms of filter behavior`
This sentence seems a bit confusing to me. Let's sync to find out if the phrasing can be improved.

Author: Removed: 
```
while also aligning the Modify Workflow functionality with the Batch Operation API in terms of filter behavior.
```
it does not add any crucial information while also confusing.

[process-instance-modification]: {{< ref "/webapps/cockpit/bpmn/process-instance-modification.md#perform-a-batch-modification" >}}

> ## Saved Filters No Longer Shared Between Modify Operation and Other Filters

Reviewer: 💭 In the effort to make the title as precise as possible, i find it confusing (too much info + repeating word).
We are already in the "Process Instance Modification Filter Changes" so i guess a more generic title is sufficient.
The user can still read and learn more about the context in detail.

```suggestion
## Saved Filters are Now Isolated
```

> When filtering process instances, users have the option to save filters for future use. Previously, saved filters were incorrectly shared between the Modify Operation and other pages. With this update, filters saved in the Modify Process Instance workflow are now isolated and will not affect filters on other pages, and vice versa.

Reviewer: ```suggestion
Filters saved in the Modify Process Instance workflow were previously shared across other pages. With this update, the filters of Modify Process Instance workflow and other pages are isolated and independent of each other.
```

> IMPORTANT: After migrating to the new version of Camunda, no saved filters will be available in the Modify Process Instance workflow. Users will need to re-enter and save these filters following the migration. Please notify your users before the migration.

Reviewer: 💭 What about adding the important note in a `note` formatting as we do in other places of the docs? I use the title "Heads-up!" in my suggestion with "warning" class as we do in other warnings (e.g see [here](https://docs.camunda.org/manual/7.21/user-guide/process-engine/history/history-cleanup/#:~:text=Heads%20Up!,they%20were%20reported.))
```suggestion
{{< note title="Heads-up!" class="warning" >}}
After migrating to the new version of Camunda, no saved filters will be available in the Modify Process Instance workflow. Users will need to re-enter and save these filters following the migration. Please notify your users before the migration.
{{< /note >}}
```
